examples/25_weave_render/src/entries.tsx

```diff
@@ -0,0 +1,57 @@
+import { lazy } from 'react';
+import { createPages } from 'waku';
+import type { PathsForPages } from 'waku/router';
+import FooPage from './components/FooPage';
+import BarPage from './components/BarPage';
+import BarLayout from './components/BarLayout';
+import FooLayout from './components/FooLayout';
+
+// The use of `lazy` is optional and you can use import statements too.
```

Comment:
I think we can remove this lazy usage from the examples.

Reply:
I see you merged it, should I make a second PR to remove this?

Reply:
Yeah, please go ahead. I mean from all examples.